New post: Hugging Face and IBM (#1130)
huggingface/blog: huggingface-and-ibm.md

---
title: "Hugging Face and IBM partner on watsonx.ai, the next-generation enterprise studio for AI builders"
thumbnail: /blog/assets/144_ibm/01.png
authors:

> - user: juliensimon

Reviewer: any co-authors from the hf.co/ibm org?

Reviewer 2: Reminder to add the post entry to `_blog.yml` in the root of the repo :)

Author: Sure. I always do it at the last minute to avoid conflicts.

---

# Hugging Face and IBM partner on watsonx.ai, the next-generation enterprise studio for AI builders

<!-- {blog_metadata} -->
<!-- {authors} -->

<kbd>
  <img src="assets/144_ibm/01.png">

> </kbd>

Reviewer 2: Interesting! What does putting the image inside `kbd` achieve? (Just curious).

Author: It adds a border around the image (see https://github.com/juliensimon/blog/blob/ibm/huggingface-and-ibm.md). That doesn't appear on our blog, for some reason.

All hype aside, it's hard to deny the profound impact that AI is having on society and businesses. From startups to enterprises to the public sector, every customer we talk to is busy experimenting with large language models and generative AI, identifying the most promising use cases, and gradually bringing them to production. 

> The #1 thing that customers tell us is that no single model will rule them all. They now understand the value of building the best model for each use case to maximize its relevance on company data while optimizing the compute budget. Of course, privacy and intellectual property are also top concerns, and customers want to ensure they maintain complete control.

Reviewer 2: ```suggestion
The #1 comment we get from customers is that no single model will rule them all. They understand the value of building the best model for each use case to maximize its relevance on company data while optimizing the compute budget. Of course, privacy and intellectual property are also top concerns, and customers want to ensure they maintain complete control.
```

2. **Pre-trained models**: [hundreds of thousands](https://huggingface.co/models) of pre-trained models are just a click away. You can discover and test them directly on [Hugging Face](https://huggingface.co) and quickly shortlist the promising ones for your projects.
3. **Open-source libraries**: the Hugging Face [libraries](https://huggingface.co/docs) let you download pre-trained models with a single line of code, and you can start experimenting with your data in minutes. From training to deployment to hardware optimization, customers can rely on a consistent set of community-driven tools that work the same everywhere, from their laptops to their production environment.

In addition, our cloud partnerships let customers use Hugging Face models and libraries at any scale without worrying about provisioning infrastructure and building technical environments. This makes it much easier to get high-quality models out the door at a rapid pace without having to reinvent the wheel.

> Following up on our collaboration with AWS on Amazon SageMaker and Microsoft on Azure Machine Learning, we're thrilled to work with none other than IBM on their new AI studio, [watsonx.ai](https://www.ibm.com/products/watsonx-ai). [watsonx.ai](http://watsonx.ai) is the next-generation enterprise studio for AI builders to train, validate, tune, and deploy both traditional ML and new generative AI capabilities, powered by foundation models.

Reviewer 2: ```suggestion
Following up on Hugging Face collaboration with AWS on Amazon SageMaker, and with Microsoft on Azure Machine Learning, we're thrilled to work with none other than IBM on their new AI studio, [watsonx.ai](https://www.ibm.com/products/watsonx-ai). [watsonx.ai](http://watsonx.ai) is the next-generation enterprise studio for AI builders to train, validate, tune, and deploy both traditional ML and new generative AI capabilities, powered by foundation models.
```

Just to remove any potential ambiguity, as the post looks like a joint announcement.

IBM decided that open source should be at the core of watsonx.ai. We couldn't agree more! Built on [RedHat OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift), watsonx.ai will be available in the cloud and on-premise. This is excellent news for customers who cannot use the cloud because of strict compliance rules or are more comfortable working with their confidential data on their infrastructure. Until now, these customers often had to build their in-house ML platform. They now have an open-source off-the-shelf alternative deployed and managed using standard DevOps tools.

> Under the hood, watsonx.ai also integrates many Hugging Face open-source libraries, such as [transformers](https://github.com/huggingface/transformers) (100k+ GitHub stars!), [accelerate](https://github.com/huggingface/accelerate), [peft](https://github.com/huggingface/peft) and our [Text Generation Inference](https://github.com/huggingface/text-generation-inference) server, to name a few. We're happy to partner with IBM and to collaborate on the watsonx AI and data platform so that Hugging Face customers can work natively with their Hugging Face models and datasets to multiply the impact of AI across businesses.

Reviewer 2: A bit too long-winded maybe? How about something like

```suggestion
Watsonx.ai already incorporates many Hugging Face open-source libraries, including [transformers](https://github.com/huggingface/transformers) (100k+ GitHub stars!), [accelerate](https://github.com/huggingface/accelerate), [peft](https://github.com/huggingface/peft), and our optimized [Text Generation Inference](https://github.com/huggingface/text-generation-inference) server. By partnering with IBM and further collaborating on the watsonx AI and data platform, we enable Hugging Face customers to seamlessly leverage their existing models and datasets, amplifying the impact of AI across businesses.
```

> IBM also developed its slate of Large Language Models, and we will work with their team to open-source them and make them easily available in Hugging Face.

Reviewer 2: ```suggestion
In addition, IBM has also developed its own collection of Large Language Models, and we will work with their team to open-source them and make them easily available in the Hugging Face Hub.
```
